contents/blog/open-source-stack-for-engineers.md

```diff
@@ -0,0 +1,160 @@
+---
+date: 2022-08-11
+title: The best open source software for engineers
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+author: ["joe-martin"]
+categories: ["Open Source", "Guides"]
+---
+
+At PostHog, we believe an open-source approach doesn’t just lead to greater growth; it also leads to better products. That’s what inspired us to make our platform open-source, to adopt [a transparent company culture](/careers) and also why we try to use open source software wherever we can in our stack. 
+
+We think the open source approach is best because it forces teams to be transparent, both in their decision making and also in their implementation. Then, because other teams have full visibility of the code, they can build on it and make it even better, faster and _stronger_. Often, they can do this for free.  
+
+Of course, PostHog isn’t the only open source software out there. In fact, there’s such variety that it’s possible to build an entirely open source stack — and here are some of our favourite open source alternatives for engineers.
+
+> Not an engineer? Find other [open source alternatives on the PostHog blog](/blog/categories/open-source)!
+
+## 1. [PostHog](/pricing)
+
+![PostHog - best open source analytics for engineers](../images/blog/gdpr-compliant-analytics/posthog-gdpr-compliant.png)
+
+- **Useful for:** Product analytics, session recording, feature flags<br/>
+- **Alternative to:** [Amplitude](/blog/posthog-vs-amplitude), [Mixpanel](/blog/why-i-ditched-google-analytics-for-posthog), [Matomo](/blog/posthog-vs-matomo)<br/>
+- **License:** [MIT](https://en.wikipedia.org/wiki/MIT_License)<br/>
+
+Of course, PostHog may not be the _only_ open source software useful to engineers, but it’s our (completely biased) favorite. 
+
+PostHog’s all-in-one product analytics platform is a direct alternative to expensive, proprietary tools such as Amplitude, Mixpanel or Heap. It’s entirely self-serve, can be self-hosted or deployed in the cloud, and offers everything from funnel analytics and path analysis to cohort creation and user tracking. Best of all, because PostHog can be deployed on-prem, it’s more suitable for [teams worried about regulatory compliance](https://isgoogleanalyticsillegal.com/). 
```

Comment:
```suggestion
PostHog’s all-in-one product analytics platform is a direct alternative to expensive, proprietary tools such as Amplitude, Mixpanel or Heap. It’s entirely self-serve, can be self-hosted or deployed in the cloud, and offers everything from funnel analytics and path analysis to cohort creation and user tracking. Best of all, because PostHog can be deployed on-prem, it’s more suitable for teams worried about [GDPR compliance](/docs/privacy/gdpr-compliance) and [HIPAA compliance](/docs/privacy/hipaa-compliance).
```